bridge/svix-bridge/src/webhook_receiver/mod.rs

```diff
@@ -181,5 +197,206 @@ async fn transform(
     }
 }
 
+struct SvixEventsPoller {
+    name: String,
+    input_opts: PollerInputOpts,
+    transformation: Option<TransformationConfig>,
+    transformer_tx: Option<TransformerTx>,
+    svix_client: Svix,
+    output: Arc<Box<dyn ReceiverOutput>>,
+}
+
+#[async_trait]
+impl PollerInput for SvixEventsPoller {
+    fn name(&self) -> &str {
+        &self.name
+    }
+
+    fn set_transformer(&mut self, tx: Option<TransformerTx>) {
+        self.transformer_tx = tx;
+    }
+
+    async fn run(&self) {
+        run_inner(self).await
+    }
+}
+
+impl PollerReceiverConfig {
+    async fn into_poller_input(
+        self,
+        transformer_tx: TransformerTx,
+    ) -> std::io::Result<Box<dyn PollerInput>> {
+        let svix_client = self
+            .input
+            .svix_client()
+            .expect("only one poller type; svix client required");
+        let name = self.name.clone();
+        let input_opts = self.input.clone();
+        let transformation = self.transformation.clone();
+        let output = Arc::new(
+            self.into_receiver_output()
+                .await
+                .map_err(|e| std::io::Error::new(std::io::ErrorKind::Other, e))?,
+        );
+        Ok(Box::new(SvixEventsPoller {
+            name,
+            input_opts,
+            transformation,
+            transformer_tx: Some(transformer_tx.clone()),
+            svix_client,
+            output,
+        }))
+    }
+}
+
+async fn run_inner(poller: &SvixEventsPoller) -> ! {
+    const MIN_SLEEP: Duration = Duration::from_millis(10);
+    const MAX_SLEEP: Duration = Duration::from_secs(300);
+    const NO_SLEEP: Duration = Duration::ZERO;
+    let mut sleep_time = NO_SLEEP;
+
+    let PollerInputOpts::SvixEvents {
+        app_id,
+        subscription_id,
+        ..
+    } = &poller.input_opts;
+
+    let mut iterator = None;
+
+    loop {
+        tracing::trace!(app_id, subscription_id, "polling `/events`");
+        match poller
+            .svix_client
+            .message()
+            .events(V1MessageStreamParams {
```

Comment:
The new "subscription" endpoint is not yet available - this will need to switch as soon as it is.